roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_priorities.py

```diff
@@ -91,7 +91,7 @@ def run(self, terms, variables=None, zones_enabled=True, short_version=None,
                 {'name': 'InterPodAffinityPriority', 'weight': 1}
             ])
 
-        if short_version in ['3.5', '3.6']:
+        if short_version in ['3.5', '3.6', '3.7']:
```

Comment:
@derekwaynecarr or @aveshagarwal are there new scheduler predicates and priorities in 3.7 that we need to reconfigure defaults for?